syntax = "proto3";

message validate_login_request {
	uint64 account_id = 1;
	uint32 server_id = 2;
}